esmvaltool/interface_scripts/messaging.ncl

```diff
@@ -79,6 +79,11 @@ begin
     if (ismissing(curr_trace_indent)) then
         print("info: " + output_string)
     else  ; Full info output
+        ;; Initialize file that keeps track of indent on first run.
+        if .not. isfilepresent_esmval(curr_trace_indent) then
+            asciiwrite(curr_trace_indent, 0)
+        end if
+
```

Comment:
This is a feature used by the NCL logging, which shall be revised in the future.

Reply:
The file needed to be created by the diagnostic script 'launcher'. I think it is better if NCL takes care of creating it's own temporary files, that's why I moved the code for creating it, if it doesn't exist yet, to the NCL script that uses it. Does this change affect NCL logging in an undesired way?